src/qibolab/instruments/qblox/cluster.py

```diff
@@ -0,0 +1,150 @@
+""" Qblox instruments driver.
+
+Supports the following Instruments:
+    Cluster
+    Cluster QRM-RF
+    Cluster QCM-RF
+    Cluster QCM
+Compatible with qblox-instruments driver 0.9.0 (28/2/2023).
+It supports:
+    - multiplexed readout of up to 6 qubits
+    - hardware modulation, demodulation, and classification
+    - software modulation, with support for arbitrary pulses
+    - software demodulation
+    - binned acquisition
+    - real-time sweepers of
+        - pulse frequency (requires hardware modulation)
+        - pulse relative phase (requires hardware modulation)
+        - pulse amplitude
+        - pulse start
+        - pulse duration
+        - port gain
+        - port offset
+    - multiple readouts for the same qubit (sequence unrolling)
+    - max iq pulse length 8_192ns
+    - waveforms cache, uses additional free sequencers if the memory of one sequencer (16384) is exhausted
+    - instrument parameters cache
+    - safe disconnection of offsets on termination
+
+The operation of multiple clusters simultaneously is not supported yet.
+https://qblox-qblox-instruments.readthedocs-hosted.com/en/master/
+"""
+
+from dataclasses import dataclass
+from enum import Enum
+
+from qblox_instruments.qcodes_drivers.cluster import Cluster as QbloxCluster
+
+from qibolab.instruments.abstract import Instrument, InstrumentException
+
+
+class ReferenceClockSource(Enum):
+    INTERNAL = "internal"
+    EXTERNAL = "external"
+
+
+@dataclass
+class Cluster_Settings:
+    """A class to store the settings of the Cluster instrument.
```

Comment:
```suggestion
    """Settings of the Cluster instrument.
```
It is clear that this is a `class` so we don't need to explicitly mention it in the docstring (to keep things shorter). This is true for many other cases too, but not very important.